mkdocs.yml

```diff
@@ -64,6 +67,10 @@ extra_css:
   - stylesheets/extra.css
 plugins:
   - search
+  - ezglossary:
+        section_config:
```

Comment:
do we require 4 indentations here? or 2 is enough?

Reply:
I think we don't. It works fine with two indentations locally.